app/views/users/show.html.erb

```diff
@@ -1 +1,10 @@
-<h1> User Page</h1>
\ No newline at end of file
+<h1> User Page</h1>
+<%= render partial: "shared/usercard", locals: { user: @user} %>
+<div>
+  <div>Bio</div>
+  <div>
+    <%= @user.bio %>
+</div>
+<% @posts.each do |post|%>
+  <%= render partial: "shared/postcard", locals: { post: post} %>
+<% end %>
```

Comment:
- [ ] Please use `recent_posts` method in the `user_controller` to render the user's 3 most recent posts because it's a project requirement. 

![image](https://user-images.githubusercontent.com/68796232/180581898-313468ed-ddbd-4ebd-86e0-83db6651ba3e.png)